index.html

```diff
@@ -1703,12 +1713,18 @@
 			'mapnum': {},
 			'order': priority_match([modules['I']['order'], modules['II']['order'], modules['III']['order']]),
 			'money': priority_match([modules['I']['money'], modules['II']['money'], modules['III']['money']]),
+			// create tokens
+			getNameTokens: getNameTokens,
```

Comment:
not certain if it is the best way to do this, I think that it is necessary for the function to be called inside params. if you can think of a better place for the function to be called or a better way for this to be handled in general let me know. but this does work.